src/Infrastructure.EntityFramework/EntityFrameworkCache.cs

```diff
@@ -0,0 +1,315 @@
+﻿using Bit.Infrastructure.EntityFramework.Models;
+using Bit.Infrastructure.EntityFramework.Repositories;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Bit.Infrastructure.EntityFramework;
+
+public class EntityFrameworkCache : IDistributedCache
+{
+#if DEBUG
+    // Used for debugging in tests
+    public Task scanTask;
+#endif
+    private static readonly TimeSpan _defaultSlidingExpiration = TimeSpan.FromMinutes(20);
+    private static readonly TimeSpan _expiredItemsDeletionInterval = TimeSpan.FromMinutes(30);
+    private DateTimeOffset _lastExpirationScan;
```

Comment:
I removed the `static` off this, this matches what the `SqlServerCache` implementation does. 

https://github.com/dotnet/aspnetcore/blob/091da452aab1838fd30f6de1decd921326736146/src/Caching/SqlServer/src/SqlServerCache.cs#L27